code/modules/integrated_electronics/subtypes/reagents.dm

```diff
@@ -150,4 +269,92 @@
 	flags = OPENCONTAINER | NOREACT
 	complexity = 8
 	spawn_flags = IC_SPAWN_RESEARCH
-	origin_tech = list(TECH_MATERIAL = 3, TECH_ENGINEERING = 2, TECH_DATA = 2, TECH_BIO = 2)
\ No newline at end of file
+	origin_tech = list(TECH_MATERIALS = 4, TECH_ENGINEERING = 2, TECH_DATA = 2, TECH_BIO = 2)
+
+/obj/item/integrated_circuit/reagent/storage/big
+	name = "Big reagent storage"
+	desc = "Stores liquid inside, and away from electrical components.  Can store up to 180u."
+	icon_state = "reagent_storage_big"
+	extended_desc = "This is effectively an internal beaker."
+	flags = OPENCONTAINER
+	complexity = 16
+	volume = 180
+	spawn_flags = IC_SPAWN_RESEARCH
+	origin_tech = list(TECH_MATERIALS = 3, TECH_ENGINEERING = 2, TECH_DATA = 2, TECH_BIO = 2)
+
+/obj/item/integrated_circuit/reagent/storage/scan
+	name = "reagent scanner"
+	desc = "Stores liquid inside, and away from electrical components.  Can store up to 60u.  On pulse this beaker will send list of contained reagents."
+	icon_state = "reagent_scan"
+	extended_desc = "This is effectively an internal cryo beaker."
```

Comment:
This description appears to be incorrect and a result of copy-pasting.  Let me know if I'm wrong.